src/index.html

```diff
@@ -12,7 +12,39 @@
       href="styles/index.scss"
     />
```

Comment:
The href attribute in the link tag points to a SCSS file (styles/index.scss). Browsers cannot directly interpret SCSS files, so you need to compile this SCSS file into a CSS file and link the resulting CSS file instead.